guide/16-geoenrichment-guides/part1_introduction_to_geoenrichment.ipynb

```diff
@@ -0,0 +1,972 @@
+{
```

Comment:
<p>Please mention where the "health.csv" is saved or can be fetched from.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>I have created a <code>data</code>  subdir in the guides folder. The csv file is in <code>data</code>  subdir and code should run now.</p>